Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,166 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+extension SetVisibilityOfSubtypeSublayerView {
+    /// The view model for this sample.
+    class Model: ObservableObject {
+        /// A map with a streets basemap style.
+        let map = Map(basemapStyle: .arcGISStreetsNight)
+        
+        /// A feature table for the electrical network in this sample.
+        private let featureTable = ServiceFeatureTable(url: .featureServiceURL)
+        
+        /// A subtype feature layer created from the service feature table.
+        private let subtypeFeatureLayer: SubtypeFeatureLayer
+        
+        /// The subtype sublayer of the subtype feature layer in this sample.
+        private(set) var subtypeSublayer: SubtypeSublayer?
+        
+        /// The renderer of the subtype feature layer.
+        private(set) var originalRenderer: Renderer?
+        
+        /// The status text to display to the user.
+        @Published private(set) var statusText = ""
+        
+        /// The  subtype sublayer's label definition.
+        private(set) var labelDefinition: LabelDefinition = {
+            // Make and stylize the text symbol.
+            let textSymbol = TextSymbol()
+            textSymbol.backgroundColor = .clear
+            textSymbol.outlineColor = .white
+            textSymbol.color = .blue
+            textSymbol.haloColor = .white
+            textSymbol.haloWidth = 2
+            textSymbol.size = 10.5
+            // Make a label definition and adjust its properties.
+            let labelExpression = SimpleLabelExpression(simpleExpression: "[nominalvoltage]")
+            let labelDefinition = LabelDefinition(labelExpression: labelExpression, textSymbol: textSymbol)
+            labelDefinition.placement = .pointAboveRight
+            labelDefinition.usesCodedValues = true
+            return labelDefinition
+        }()
+        
+        // The formatter used to generate strings from scale values.
+        private let scaleFormatter: NumberFormatter = {
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            numberFormatter.maximumFractionDigits = 0
+            return numberFormatter
+        }()
+        
+        /// A Boolean value indicating whether the settings should be presented.
+        @Published var isShowingSettings = false
+        
+        /// A Boolean value indicating whether to show the subtype sublayer.
+        @Published var showsSublayer = true
+        
+        /// A Boolean value indicating whether to show the subtype sublayer's renderer.
+        @Published var showsOriginalRenderer = true
+        
+        /// The current scale of the map.
+        @Published var currentScale: Double = .zero
```

Comment:
It seems the view doesn't depend on this value, so no need to be published? However if the format style suggestion is applied, then the value can be formatted in place.